packages/plugin-js-packages/src/lib/js-packages-plugin.unit.test.ts

```diff
@@ -16,7 +16,8 @@ describe('jsPackagesPlugin', () => {
     ).resolves.toStrictEqual(
       expect.objectContaining({
         slug: 'js-packages',
-        title: 'Plugin for JS packages',
+        title:
```

Comment:
Could we get a shorter title and more text in  the description? This would otherwise look odd in a tooltip I think

Reply:
Sure! Any ideas?

Reply:
JS Package Versions?

Reply:
Yes, title should be shorter and `Plugin for ` is superfluous. I would suggest that in most cases, the `title` is simply the non-slugified form of the `slug` and package name (without `plugin`). So `JS Packages` would be the obvious candidate. I wouldn't go with `JS Package Versions`, because outdated versions is only one of the features, security audit is another big part of it.

Reply:
JS Packages it is, thank you for the ideas.